dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -992,7 +1011,8 @@ def check_raises(d, p, func):
         check_raises(dds, pds, "std")
         check_raises(dds, pds, "var")
         check_raises(dds, pds, "sem")
-        check_raises(dds, pds, "mean")
+        if not (PANDAS_GT_0250 and is_datetime64_ns_dtype(pds.dtype)):
+            check_raises(dds, pds, "mean")
```

Comment:
Will be useful for knowing when to remove the `PANDAS_GT_0250` check 

```suggestion
            # pandas 0.25 added DatetimeIndex.mean. We need to follow
            check_raises(dds, pds, "mean")
```